sound/soc/sof/topology.c

```diff
@@ -804,6 +817,70 @@ static int sof_connect_dai_widget(struct snd_soc_component *scomp,
 	return 0;
 }
 
+/* LBM kcontrol, used to control loopback mode on SSP */
+static int sof_dai_add_lbm_kcontrol(struct snd_soc_component *scomp,
```

Comment:
This is completely generic code except for the strings and provate data. These should be passed in as args.

Reply:
Thought about that.
Will do that.